hs_core/views/discovery_json_view.py

```diff
@@ -71,5 +72,5 @@ def form_valid(self, form):
             # return JSON response
 
             return HttpResponse(the_data, content_type='application/json')
-        else:
-            return HttpResponse(json.dumps('[]'), content_type='application/json')
+        # else:
```

Comment:
@alvacouch What will happen here if there is no GET data in the request? a 500 error?

Reply:
You tell me what should happen. I returned nothing. Zhaokun disagreed.

Reply:
Well we should explicitly return _something_, we can't just let the system decide what gets sent along. I say just uncomment these for now and keep the one-line fix.